charts/bria/templates/bria-cm.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "bria.fullname" . }}-config
+data:
+  bria.yml: |-
```

Comment:
this won't work. Need a translation from `camelCase` <- idiomatic helm to `snake_case` <- expected in bria.
See how this is done in stablesats.